### DISCUSSION TITLE
Domain renewal issues should be resolved in a private support case not public forums.

### DISCUSSION THREAD

**Discussion by taylorjdawson** [vercel/community#3864] [Help]
### Summary

For expired domains Vercel asks that you contact support immediately. However, when select Domains & DNS as a `Problem Area` in the drop down, it disables the ability to create a support case and directs you to this forum. 
However, if your domain expires that is a very individual customer issue that shouldn't be discussed publicly as the resolution of that issue is of no benefit to anyone else in the community. Further more there could be privacy issues with sharing which domain that you owned has expired.

There should be an option to create a case for expired domains.

### Steps to Reproduce

- Go to https://vercel.com/help
- Select `Domains & DNS` in the `Problem Area` selection options
<img width="819" alt="Screenshot 2023-08-31 at 10 50 16" src="https://github.com/vercel/community/assets/29665732/0adba573-2612-49b8-9cbf-7e96adc3ba47">

**Comment by amyegan** [comment]
Hey @taylorjdawson. Sorry for the inconvenience on this. Please [submit a case](https://vercel.com/help#issues) to the Customer Success Team under the `Billing` problem area and let me know the case number once you have it. I'll flag it with the team to make sure they see it right away 🙏
